subprojects/logging/src/main/java/org/gradle/internal/logging/sink/OutputEventRenderer.java

```diff
@@ -329,6 +352,10 @@ public void onOutput(OutputEvent event) {
                 return;
             }
             this.maxWorkerCount.set(newMaxWorkerCount);
+        } else if (event instanceof EndOutputEvent) {
```

Comment:
❓ how are we handling the case you mentioned where these events are not generated when using the TAPI?